include/control_toolbox/pid_ros.hpp

```diff
@@ -122,6 +122,19 @@ class CONTROL_TOOLBOX_PUBLIC PidROS
    */
   double computeCommand(double error, rclcpp::Duration dt);
 
+  /*!
+   * \brief Set the PID error and compute the PID command with nonuniform
+   * time step size. This also allows the user to pass in a precomputed
+   * derivative error.
+   *
+   * \param error Error since last call (error = target - state)
+   * \param error_dot d(Error)/dt since last call
+   * \param dt Change in time since last call in seconds
+   *
+   * \returns PID command
+   */
+  double computeCommand(double error, double error_dot, rclcpp::Duration dt);
```

Comment:
Do we need additional method for this? Why not always calculate `error_dot` in the `computerCommand` method and use only one of them? I understand there are two underlaying methods, but why?

Can you give more information about this change?

Also, I propose to add this change separately into another PR because it is unrelated to the PR title and paramter feature.

Reply:
Because I'm trying to used IMU data to do the PID control. And I'm using calculated angle from IMU to compute the `error` and using angular velocity directly read from IMU as the `error_dot`.

I will open another PR to commit this change. Thank you for your reply.